mrmustard/lab_dev/circuit_components.py

```diff
@@ -421,6 +447,10 @@ class DualView(CircuitComponent):
         component: The circuit component to take the view of.
     """
 
+    def __getattr__(self, name):
+        r"""send calls to the component"""
+        return getattr(self._component, name)
```

Comment:
:+1:

Reply:
In retrospective, maybe it would have been better to have a `View` class that adjoint and dual can inherit from, to avoid repeated code

Reply:
good idea. I'll put it in the todos